pom.xml

```diff
@@ -338,7 +339,7 @@
         <dependency>
             <groupId>net.snowflake</groupId>
             <artifactId>snowflake-ingest-sdk</artifactId>
-            <version>2.3.0</version>
+            <version>3.0.0</version>
```

Comment:
Please align `pom.xml` with `pom_confluent.xml`.

Reply:
Sry, I forgot about `pom_confluent.xml`, however I must revert back to 2.3.0. , 3.0.0 is not yet available.